tests/integ/test_local_mode.py

```diff
@@ -349,12 +349,12 @@ def test_local_processing_sklearn(sagemaker_local_session_no_local_code, sklearn
 
     job_description = sklearn_processor.latest_job.describe()
 
-    assert len(job_description["ProcessingInputs"]) == 2
+    assert len(job_description["ProcessingInputs"]) == 3
```

Comment:
I’m trying to understand why there is now one more ProcessingInputs?

Reply:
@verdimrc @athewsey any updates?

Reply:
Hi, apology, my reply went to the PR comment instead of this area.

The test for the original sklearn processor creates these input channels:
1. `/opt/ml/processing/inputs` => input data
2. `/opt/ml/processing/input/code` => to place `dummy_script.py` which is the container entrypoint.

The new sklearn processor creates these input channels:
1. `/opt/ml/processing/inputs` => input data
2. `/opt/ml/processing/input/code` => to place `sourcedir.tar.gz` which contains `dummy_script.py`
3. `/opt/ml/processing/input/entrypoint` => to place bootstrap logic `runproc.py` which is the container entrypoint.